tests/snmpsim/cyberpower.snmprec

```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|Power Manager
+1.3.6.1.2.1.1.2.0|6|.1.3.6.1.4.1.3808.1.1.3
```

Comment:
To keep things the same, drop the leading `.` from `.1.3.6...`: `1.3.6.1.4.1.3808.1.1.3`